doc/direction.md

```diff
@@ -0,0 +1,65 @@
+# Minetest Direction Document
+
+---------------------------
+
+## 0. Preface
+
+### 0.1. Introduction
+
+This document attempts to set out Minetest's philosophy, aims, and objectives.
+
+Please note that Minetest is developed by a community, and not everyone in the community
+holds the same views as to what Minetest should be.
+
+### 0.2. Definitions
+
+* `Minetest` is the game engine.
+* `Minetest Game` is the default game which comes with the engine.
+* The `Minetest Project` refers to the combination of the above, the development team, and supporting tools.
+
+### 0.3. Other documents
+
+celeron55 has published his opinions on the development of Minetest in these places:
+
+* What is Minetest? - http://c55.me/blog/?p=1491
+* Roadmap - https://forum.minetest.net/viewtopic.php?t=9177
+* A clear mission statement for Minetest is missing - https://github.com/minetest/minetest/issues/3476#issuecomment-167399287
+
+Also see the roadmaps and manifestos of core developers on the forums: https://forum.minetest.net/viewforum.php?f=7
+
+---------------------------
+
+## 1. Mission Statement
+
+Minetest provides both a game engine, which allows the creation of voxel games using Lua, and
+a content distribution platform. Players can connect to servers with their own gameplay, and
```

Comment:
> their own gameplay

I think it should be reprased to emphasise the fact that "servers has own gameplay" not that "players has own gameplay".

Reply:
I agree, the wording is ambiguous at present.